pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -52,6 +52,7 @@ const ProviderName = "aws"
 
 // The tag name we use to differentiate multiple logically independent clusters running in the same AZ
 const TagNameKubernetesCluster = "KubernetesCluster"
+const TagNameKubernetesNode = "KubernetesNode"
```

Comment:
Can we make this `kubernetes.io/node` or `kubernetes.io/nodename`.  I screwed up before by not prefixing KubernetesCluster, and eventually I'm going to have to fix it...